applications/FluidDynamicsApplication/custom_strategies/strategies/gear_scheme.h

```diff
@@ -727,6 +755,11 @@ class GearScheme : public Scheme<TSparseSpace, TDenseSpace>
         rModelPart.GetCommunicator().AssembleCurrentData(DIVPROJ);
         rModelPart.GetCommunicator().AssembleCurrentData(ADVPROJ);
 
+        // Correction for periodic conditions
+        if (mrPeriodicIdVar.Key() != 0) {
```

Comment:
can it still be zero?

Reply:
@pooyan-dadvand what is the key of the static object for a variable type now? I understood it should still be zero.